pdl/protocol.pdl

```diff
@@ -8,6 +8,15 @@ domain Recording
   # Globally unique identifier for a recording.
   type RecordingId extends string
 
+  type RecordingData extends object
```

Comment:
Main TODO is probably this. We wanted this definition to move to `Internal`.

Reply:
Ah right, will do